Mathlib/Tactic/Differentiability.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2023 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll, Adomas Baliuka
+-/
+import Mathlib.Tactic.Differentiability.Init
+import Mathlib.Algebra.Group.Defs
+
+/-!
+# Differentiability
+
+We define the `differentiability` tactic using `aesop`.
+The approach is copied from how the tactic `continuity` is implemented.
+-/
+
+attribute [aesop (rule_sets [Differentiable]) unfold norm] Function.comp
+
+-- Issue: Large natural powers lead to timeouts.
+-- This is due to copied over (from `continuity`) settings of tactic.
+-- I'm not sure why those options are used.
+attribute [aesop (rule_sets [Differentiable]) unfold norm] npowRec
```

Comment:
`unfold` on `npowRec` is no longer accepted by the latest `aesop`.

On the `continuity` and `measurability` tactics I've just added the note:
```
-- FIXME: `npowRec` is an internal implementation detail,
-- and `aesop` certainly should not know about it.
-- If anyone is working on the `continuity` tactic, please try to fix this!
```

I think this should be addressed here before we commit further to this mistake! :-)